recipes/deepac/meta.yaml

```diff
@@ -32,7 +32,7 @@ requirements:
     - statsmodels >=0.11
     - shap >=0.35
     - tensorboard >=2.1
-    - tensorflow >=2.1
+    - tensorflow-gpu >=2.1
```

Comment:
is this now also working on CPU? If not should we add something to the descripton?

Reply:
Yes, it works on CPU as well (tested on a machine without any GPUs before committing). It used to be a problem but aparently not anymore, so I was considering requiring tf-gpu for convenience (since in practice this is the "default" usage). But the more I think about it, the more it looks like fixing somethin that ain't broken at a cost of installing optional gpu-related dependencies (and a different tf build). Maybe leaving the gpu-enabled installation to the user (as it was before) is the more stable and efficient solution after all.